editor/components/block-list/style.scss

```diff
@@ -345,6 +412,10 @@
 		position: absolute;
 		top: 0;
 		padding: 0;
+
+		/* Necessary for drag indicator */
```

Comment:
Necessary in what way?

Reply:
cc @chriskmnds

Reply:
It's for showing the grab handle when hovering over the area.